### DISCUSSION TITLE
"Up Next" list is filled with political videos

### DISCUSSION THREAD

**Discussion by ajgringo619** [FreeTubeApp/FreeTube#5588] [General]
How does FT determine what videos are shown in the "Up Next" list? Lately I've been getting more than my fair share of political videos, things that I never watch either here or on YouTube. Is there a way to permanently block these?

**Comment by absidue** [comment]
> How does FT determine what videos are shown in the "Up Next" list?

FreeTube just displays what it gets from YouTube. You have to keep in mind that when you are on YouTube itself, YouTube tracks your every move so they can customize the Up Next feed/recommended videos for you personally. As FreeTube's primary goal is to be a privacy friendly YouTube app, it tries it's best to avoid that tracking. So all YouTube can go off is what it thinks is similar to that video or maybe popular things in your area, I'm just guessing here though because YouTube doesn't tell anyone outside of YouTube how their algorithms work.

**Comment by ajgringo619** [comment]
OK, that makes sense. Considering that all of the linked videos are on one side of the political spectrum, this is obviously another way for Google to influence the US electorate. This is probably the main reason I use FT - no candidate pandering in the middle of videos.

**Reply by absidue** [reply]
If you don't want to see the recommended videos anymore you can turn on `Hide Recommended Videos` in the `Distraction Free Settings` (it's in the `Watch Page` subsection)

**Reply by ajgringo619** [reply]
Just what I needed - thanks!
